README.md

```diff
@@ -89,7 +89,6 @@ $ git add .
 
 ### Contributing
 
-- running test locally
-
+- running DNS test locally: set these env before running test: `GCORE_DNS_API=https://api.gcore.com/dns;GCORE_PASSWORD=xxx;GCORE_USERNAME=xxx;TF_ACC=true;TF_LOG=DEBUG`
```

Comment:
but one can not run tests locally

Reply:
why not? '__')
![image](https://github.com/G-Core/terraform-provider-gcore/assets/1061610/c256a8e7-0f93-49a2-a159-d5441773c4ba)

Reply:
but it's not locally